script.js

```diff
@@ -0,0 +1,231 @@
+console.log(isEven);
+
+let isLoggedIn = false;
+let user = isLoggedIn ? "Feel free to check our new offers" : "Please log in first";
+console.log(isLoggedIn);
+console.log(user);
+
+let pricePerUnit = 20;
+let quantity = 5;
+let availability = true;
+let totalPrice = availability ? pricePerUnit * quantity : "N/A";
+console.log(pricePerUnit);
+console.log(quantity);
+console.log(availability);
+console.log(totalPrice);
+
+let currentYear = new Date().getFullYear();
+let leapYear = currentYear % 4 === 0 || (currentYear % 400 === 0 && currentYear % 100 === 0) ? "A Leap Year" : "Not A Leap Year";
+console.log(currentYear);
+console.log(leapYear);
+
+// Switch //
+
+let countryCode = "MD";
+switch (countryCode) {
+    case "MD":
+        console.log("Moldova");
+        break;
+    case "IN":
+        console.log("India");
+        break;
+    case "IT":
+        console.log("Italy");
+        break;
+    case "US":
+        console.log("United States");
+        break;
+    case "FR":
+        console.log("France");
+        break;
+        default:
+    console.log("Unknown country");
+}
+
+let points = "65";
+switch (true) {
+    case points >=90:
+        console.log("A");
+        break;
+    case points >=80:
+        console.log("B");
+        break;
+    case points >=70:
+        console.log("C");
+        break;
+    case points >=60:
+        console.log("D");
+        break;
+    case points <=59:
+        console.log("F");
+        break;
+    default:
+        console.log("Not a valid grade");
+}
+
+// Function //
+
+//tellFortune exercise //
+
+function tellFortune (n, z, y, x) {
+    return (`You will be a ${x} in ${y} and will get married to ${z} with ${n} kids`);
+}
+
+let person1 = tellFortune (2, "John", "London", "CFO");
+console.log(person1);
+
+let person2 = tellFortune (3, "Jack", "Madrid", "PR Specialist");
+console.log(person2);
+
+let person3 = tellFortune (4, "Bob", "Paris", "CTO");
+console.log(person3);
+
+// Dog's age exercise //
+
+function calculatedDogAge (humanYears, coefficient=7) {
+    let dogAge = humanYears*7;
+    console.log(`Your dog is ${dogAge} years old in dog years`);
+};
+calculatedDogAge (4);
+calculatedDogAge (6);
+calculatedDogAge (1);
+
+// Calculation Supply //
+
+let calculationSupply = (age, foodQuantity, maxAge=100) => {
+let result = (maxAge-age)*foodQuantity*365;
+console.log(`You will need ${result} kg of food to survive until you are ${maxAge} years old`);
+};
+calculationSupply (50, 1, 80);
+calculationSupply (30, 2, 90);
+calculationSupply (15, 0.5);
+
+// HW-4 //
+
+// Ex-1 //
+
+let fruits = [];
+fruits.push("apple","banana","orange");
+console.log(fruits.length);
+console.log(fruits);
+
+// Ex-2 //
+
+let numbers = [1, 2, 3, 4, 5];
+console.log(numbers);
+
+let removedNo1 = numbers.pop();
+console.log(numbers);
+
+let removedNo2 = numbers.shift();
+console.log(numbers);
+
+// Ex-3 //
+
+let colors = ["red", "green", "blue", "yellow"];
+let index = colors.indexOf ("green");
```

Comment:
don't add spaces between method name and (

Reply:
got it :)